src/server/server.js

```diff
@@ -252,7 +252,7 @@ SimulationServer.prototype._streamAppHostHtml = function (request, response) {
             send(request, filePath, {
                 transform: function (stream) {
                     return stream
-                        .pipe(replaceStream(/(<\s*head[^>]*>)/, '$1' + scriptTags))
+                        .pipe(replaceStream(/(<\s*head[\s]*>)/, '$1' + scriptTags))
```

Comment:
This will only match a `<head>` tag that contains "head" and white-space, but nothing else. So if the head tag contains attributes, for example, this will fail.

One possibility would be to take the original regex and include a word boundary after "head": `/(<\s*head\b[^>]*>)/`